5/JPA/LearningJava/HELP.md

```diff
@@ -0,0 +1,9 @@
+# Getting Started
```

Comment:
Please review this file as well as it's not necessary

Reply:
ok